auto_nag/scripts/regression_without_regressed_by.py

```diff
@@ -0,0 +1,207 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import dateutil.parser
+from libmozdata.bugzilla import Bugzilla, BugzillaUser
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag import utils
+
+
+class RegressionWithoutRegressedBy(BzCleaner):
+    def __init__(self):
+        super(RegressionWithoutRegressedBy, self).__init__()
+
+    def description(self):
+        return 'Regressions without regressed_by and some dependencies'
+
+    def handle_bug(self, bug, data):
+        bugid = bug['id']
+        deps = set(bug['blocks']) | set(bug['depends_on'])
+        assignee = bug['assigned_to_detail']
+        if utils.is_no_assignee(assignee['email']):
+            assignee = None
+
+        data[str(bugid)] = {
+            'deps': deps,
+            'assignee': assignee,
+            'creator': bug['creator_detail'],
+            'creation': dateutil.parser.parse(bug['creation_time']),
+        }
+        return bug
+
+    def filter_bugs(self, bugs):
+        all_deps = set()
+        dep_bug_creation = {}
```

Comment:
Instead of `dep_bug_creation`, can we call this something like `regression_bug_creation`? At first I was confused because I thought this was a map of the creation times of the dependencies